src/sentry/search/eap/span_columns.py

```diff
@@ -619,6 +620,19 @@ def count_processor(count_value: int | None) -> int:
             )
         ],
     ),
+    "http_response_rate": FunctionDefinition(
+        internal_function=Function.FUNCTION_UNSPECIFIED,
+        default_search_type="percentage",
+        arguments=[
+            ArgumentDefinition(
+                argument_types={
+                    "string"
+                },  # TODO - this should be an integer, but `resolve_attribute` returns a string
+                default_arg=None,  # TODO - this should only accept 2,3,4,5
+            )
+        ],
+        formula_resolver=CUSTOM_FUNCTION_RESOLVER["http_response_rate"],
+    ),
```

Comment:
🤔 should we make a separate `SPAN_FORMULA_DEFINITIONS`? ie. if we're defining formulas in `span_formulas` maybe we just import the definitions from there and put it into the definitions directly?